releasenotes/notes/add-max-message-size-bytes-config-7e943bc4966d0dec.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    `max_message_size_bytes` can now be configured in `logs_config`
```

Comment:
RST uses double backticks.
```suggestion
    ``max_message_size_bytes`` can now be configured in ``logs_config``
```

Also, would it make sense to mention what the setting is for?